backend/app/api/api_v1/endpoints/role.py

```diff
@@ -19,11 +19,6 @@ def get_all_role(
     Retrieve all roles.
     """
     roles = crud.role.get_multi(db, skip=skip)
-    if not roles:
-        raise HTTPException(
-            status_code=404,
-            detail="Roles not found",
-        )
```

Comment:
Is there a reason for removing this? Not sure if it's a commit issue

Reply:
think it's a commit issue, ive added it back in!